app/controllers/public/speaker/view.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+
+export default class extends Controller {
+  
```

Comment:
Codacy found an issue: [Trailing spaces not allowed.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403732)